bin/deploy

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+current_branch=$(git branch --show-current)
+
+git stash && \
+  git checkout master && \
+  git fetch origin &&
+  git pull origin master && \
+  bundle install && \
+  bundle exec rake deploy  && \
```

Comment:
On perd la possibilité de déployer sur différents environnements non ?

Reply:
Ce fichier est seulement pour la prod (y'a un tag and stuff, chose qu'on veut pas sur le reste des envs), il y a des explications dans le README pour faire des déploiements exotiques.

Reply:
Ok ça me va !